### DISCUSSION TITLE
esp32 .bin files

### DISCUSSION THREAD

**Discussion by clucia01** [awawa-dev/HyperSPI#28] [Q&A] [answered]
Where are the .bin files for flashing esp32? I have esphome flasher but no files to flash and no idea where to get them.

**Comment by awawa-dev** [comment, marked as answer]
In the release section, latest  https://github.com/awawa-dev/HyperSPI/releases/tag/v9.0.0.0

**Comment by clucia01** [comment]
after flashing the .bin for ws2812 it doesnt seem like i am getting any meaningful connection to my pi. the 1st two leds are turning on, the first white and the second yellow, but one turns on with the power cable plugged in and power off and then both when the power is turned on. the only thing i can figure is i dont have it wired right or the software is not configured correctly. for what its worth i am not using the voltage leveler. i planned to use the method of disabling the first led.

**Reply by awawa-dev** [reply]
Probably these are "phantom" lights and the led strip is not controlled by esp at all. Are you using release files or your fork? https://github.com/awawa-dev/HyperSPI/compare/master...clucia01:HyperSPI:master You are only made a change to the comment section so it doesnt affect the your built files. it's hard to judge whether the problem is the software configuration or the problem is in the hardware, especially since there is no level-shifter. Try to test HyperSerialESP32 (or WLED still powering up esp32 from the Rpi USB) first as both should be easier to start with. If everything is fine with your hardware then you can always go back to HyperSPI.

**Reply by clucia01** [reply]
i am using release files. i did the fork but realized shortly after that that was only needed for a custom config (i think). I will try the other methods and see where i get. thanks
